.rubocop_todo.yml

```diff
@@ -6,12 +6,6 @@
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-Layout/ClosingHeredocIndentation:
-  # As far as I can tell, this lint rule just doesn't work when it comes to
-  # heredoc in `it` blocks.
-  Exclude:
-    - 'test/**/*'
-
```

Comment:
I basically had to kill this one before because I think it had some buggy false positives for non-squiggly heredocs in test `it` blocks. I changed to squiggly heredocs in those tests, and it now seems to be happy so I took the exemption back out.